adapters/info.go

```diff
@@ -206,6 +206,7 @@ type BidderInfo struct {
 	Capabilities            *CapabilitiesInfo `yaml:"capabilities" json:"capabilities"`
 	AliasOf                 string            `json:"aliasOf,omitempty"`
 	ModifyingVastXmlAllowed bool              `yaml:"modifyingVastXmlAllowed" json:"-" xml:"-"`
+	DebugInfo               *config.DebugInfo `yaml:"debug,omitempty" json:"debug,omitempty"`
```

Comment:
The json must be `json:"-" xml:"-"` to play nicely with the /info/bidders endpoint.

Reply:
Right, didn't know about this, fixed